.github/workflows/release-cli.yml

```
@@ -10,10 +10,14 @@ jobs:
         rid: ['win-x64', 'osx-x64', 'linux-x64']
     env:
       NUGET_URL: https://nuget.pkg.github.com/microsoft/index.json
-      PACKAGE_VERSION: unversioned
-      PACKAGE_TEMPLATE: msgraph-cli-{0}-{1}
+      PACKAGE_VERSION: ${{github.ref_name || 'v1.0.0'}}
```

```suggestion
      PACKAGE_VERSION: ${{github.ref_name || 'v0.1.0'}}
```
v1 starts when the CLI GAs